vllm/attention/ops/prefix_prefill.py

```diff
@@ -49,6 +49,7 @@ def _fwd_kernel(
         BLOCK_M: tl.constexpr,
         BLOCK_DMODEL: tl.constexpr,
         BLOCK_N: tl.constexpr,
+        BLOCK_DMODEL_USED: tl.constexpr,
```

Comment:
This is the actual batch size that's not padded right? Maybe add a simple comment here?

Reply:
I also renamed the variables instead to `BLOCK_DMODEL` and `BLOCK_DMODEL_PADDED`